cfg/config.go

```diff
@@ -3,11 +3,97 @@
 // that will be retrieved from  .env or ./cfg/config.yml
 package cfg
 
+import (
+	"bufio"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+// Load configs from a env file & sets them in environment variables .
+func LoadEnvVar() error {
+	f, err := os.Open(".env")
+
+	if err != nil {
+		log.Printf("%s", err)
+		return err
+	}
+
+	defer func() {
+		err := f.Close()
+		if err != nil {
+			log.Printf("%s", err)
+		}
+	}()
+
+	lines := make([]string, 0)
+	scanner := bufio.NewScanner(f)
+
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+
+	if err := scanner.Err(); err != nil {
+		log.Printf("%s", err)
+		return err
+	}
+
+	for _, l := range lines {
+		pair := strings.Split(l, "=")
+		os.Setenv(pair[0], pair[1])
+	}
+	return nil
+}
+
+var (
+	serverHost = os.Getenv("SERVHOST")
```

Comment:
Non-blocker: most environment variables use names in all caps with underscores, so DB_NAME

Reply:
What does your prefix really mean, may i ask? I mean "Non-blocker" before u start the sentence.

Reply:
Thank you for the opportunity to look at it differently. I was guided by these recommendations: https://www.postgresql.org/docs/current/libpq-envars.html

Reply:
> What does your prefix really mean, may i ask? I mean "Non-blocker" before u start the sentence.

"Non-blocker" means that this comment does not block the approval, but is a recommendation

Reply:
That's what I thought. TY.

Reply:
These are environment variable names in Postgresql, but we're talking about variable names in Go, aren't we?

Reply:
To be an honest mix of uppercase and snake case closer to Python rather than Go style. Unfortunately, we have to deal with several paradigms simultaneously and looks like @vdshakh are right.

Reply:
> These are environment variable names in Postgresql, but we're talking about variable names in Go, aren't we?

You got me) Renamed

Reply:
> To be honest mix of uppercase and snake case is closer to Python rather than Go style. Unfortunately, we have to deal with several paradigms simultaneously and looks like @vdshakh are right.

My heart agreed with her recommendation, but laziness did not)